src/main/java/com/buinak/imageapi/service/ImageService.java

```diff
@@ -35,15 +39,23 @@ public Image addImage(String name, String description, MultipartFile file) {
         return imageRepository.saveAndFlush(image);
     }
 
+    public List<String> listImageUrls() {
+        final String BASE_URL =
+                ServletUriComponentsBuilder.fromCurrentContextPath().build().toUriString();
+        return imageRepository.findAll().stream()
+                .map(image -> BASE_URL + File.separator + image.getPath())
```

Comment:
Extract this to a constant which would be a template and then use String.format() on it, more clear what's going on I believe